Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM python:3.10 AS base
+
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1 \
+    PYTHONFAULTHANDLER=1 \
+    PYTHONHASHSEED=random \
+    PYTHONUNBUFFERED=1
+
+WORKDIR /build
+
+# Create virtualenv using poetry lockfile
+FROM base AS builder
+
+ENV PIP_NO_CACHE_DIR=off \
+    PIP_DEFAULT_TIMEOUT=100 \
+    POETRY_VERSION=1.6.1
+
+RUN pip install eth-ape ape-vyper
+
+COPY dropbot/ape-config.yaml ape-config.yaml
+COPY dropbot/contracts/erc721.vy contracts/erc721.vy
+
+RUN ape compile
+
+RUN ls -lah .build
+
+FROM apeworx/silverback:latest
+
+WORKDIR  /app
+
+# install and git
+USER root
+RUN apt-get -y update && apt-get -y install git
+
+# Install Necessary plugins for chaosnet environment
+RUN pip install asyncer
+
+# Install Silverback private plugin
+RUN ssh-keyscan -H github.com >> /etc/ssh/ssh_known_hosts
+RUN --mount=type=ssh,id=github_ssh_key \
+    pip install "git+ssh://git@github.com/ApeWorX/ape-silverback@main#egg=ape_silverback"
```

Comment:
Eventually try to migrate this away somehow, but fine for now

Reply:
I agree.  I just don't have an immediate solution to it.